Makefile

```diff
@@ -0,0 +1,8 @@
+init:
```

Comment:
Next time you can also look at making a make install for the dependencies :)

Reply:
Currently it's all included into the "start" cmd because npm depencies are installed and builded during the image construction !